.github/workflows/packages.yml

```
@@ -41,72 +41,75 @@ jobs:
       - name: Run tests # Run tests of all packages
         run: npx lerna exec npm run ci:lint --parallel
 
+      - name: Run spellcheck # Run spellcheck of all packages
+        run: npx lerna exec npm run ci:spellcheck --parallel
```

How do we feel about using `--if-present` flag for the `ci:spellcheck` script? This should allow to define the script only in the packages we care about the spellcheck and simply ignore the rest of the packages.
```suggestion
        run: npx lerna exec npm run ci:spellcheck --parallel -- --if-present
```